sdk/servicebus/service-bus/README.md

```diff
@@ -39,39 +39,25 @@ You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your t
-- [createFromAADTokenCredentials](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#createfromaadtokencredentials-string--applicationtokencredentials---usertokencredentials---devicetokencredentials---msitokencredentials--servicebusclientoptions-)
+- [Create using a TokenCredential](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#servicebusclient-string--tokencredential---servicebusclientoptions-)
   - This method takes the host name of your Service Bus instance and a credentials object that you need
-    to generate using the [@azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth)
+    to generate using the [@azure/identity](https://www.npmjs.com/package/@azure/identity)
     library. The host name is of the format `name-of-service-bus-instance.servicebus.windows.net`.
-  - Refer to the samples that use an [Azure account](https://github.com/Azure/azure-sdk-for-js/blob/%40azure/service-bus_1.0.0/sdk/servicebus/service-bus/samples/javascript/gettingStarted/loginWithAzureAccount.js), [interactive login](https://github.com/Azure/azure-sdk-for-js/blob/%40azure/service-bus_1.0.0/sdk/servicebus/service-bus/samples/javascript/gettingStarted/interactiveLogin.js) or [service principal](https://github.com/Azure/azure-sdk-for-js/blob/%40azure/service-bus_1.0.0/sdk/servicebus/service-bus/samples/javascript/gettingStarted/servicePrincipalLogin.js)
 
 ### Key concepts
 
-Once you have initialized the [ServiceBusClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient) class, use the below methods to create client
-objects for Queues, Topics and Subscriptions to interact with existing Service Bus entities. Please
-note that the Queues, Topics and Subscriptions should already have been created prior to using this
-library.
-
-- [createQueueClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#createqueueclient-string-)
-  - Takes the name of an existing Service Bus Queue instance, returns a QueueClient that you can use
-    to send to and receive messages from the queue.
-- [createTopicClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#createtopicclient-string-)
-  - Takes the name of an existing Service Bus Topic instance, returns a TopicClient that you can use
-    to send messages to the topic
-- [createSubscriptionClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#createsubscriptionclient-string--string-)
-  - Takes the name of existing Service Bus Topic and Subscription instances, returns a SubscriptionClient
-    that you can use to receive messages from the subscription.
+Once you have initialized the [ServiceBusClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient)
+class, use the below methods to create client objects for Queues, Topics and Subscriptions
```

Comment:
We no longer have "client objects"...
Previously these were meant to be the QueueClient, TopicClient and SubscriptionClient

Reply:
```suggestion
class, create a sender or receiver based on whether you want to send or receive messages.
```